### DISCUSSION TITLE
using dbeaver to query parquet files

### DISCUSSION THREAD

**Discussion by qiangbo** [duckdb/duckdb#3726] [General]
Hi,

I have followed the tutorial to install dbeaver and configured the driver with :memory. It seems running ok.

Then, when I try the following query:

select
    *
from
    read_parquet('C:/data/test.parquet')
limit 10;

DBeaver gives a lot of error messages.

Am I on the right path to get it running?

Thanks!

Bo

**Comment by Alex-Monahan** [comment]
Hello! 

Could you provide the error messages you are seeing? And could you also provide the parquet file, or an anonymized version of it? That helps us write tests to be sure errors do not return!

Thanks for checking out the guide!
